The-Product-Hub/iReporter
# [feature #162162217] created a home/landing page

####What does this PR do?
This PR is the homepage the user sees when he goes to the site. It gives more details about the purpose of the application.

####Description of Task to be completed?
Create a web-page that gives more details about the site.
Allow user to sign up for an account.

####How should this be manually tested?
User should be able to land on this page when the site link is clicked
He/She should then be redirected on the create an account page.
The sign up button should be working

####Any background context you want to provide?
The site should be simple and easy to navigate

####What are the relevant pivotal tracker stories?
#162162217
